#string name
---
string[] ades_list
